route/build/src/mpi_process.f90

```diff
@@ -1808,6 +1872,8 @@ SUBROUTINE mpi_comm_river_flux(pid,          &
   character(len=strLen)                   :: cmessage              ! error message from a subroutine
   real(dp),     allocatable               :: flux(:,:)             !
   real(dp),     allocatable               :: flux_local(:,:)       !
+  real(dp),     allocatable               :: mass(:,:)             ! solute mass flux in main task
```

Comment:
If this is solute, it seems like that should be reflected in the name of the variables.

Reply:
change mass to solute_mass